opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -217,6 +218,14 @@ message DoubleHistogram {
   AggregationTemporality aggregation_temporality = 2;
 }
 
+// DoubleSummary metric data are used to convey quantile summaries,
+// a Prometheus and OpenMetrics data type. These data points cannot
```

Comment:
Can we leave out the references to Prometheus and OpenMetrics? Do they add anything of value? If it is important for understanding the semantics then it is best to include a link to Prometheus and OpenMetrics that explain it.

Reply:
Making this a link is better. We want to keep the reference because as the PR explains the summary is mostly to be able to carry openmetrics/Prometheus summaries not for us to produce them

Reply:
Just to get some clarity on the ask, do you want me to add links to https://prometheus.io/ and https://openmetrics.io/ ?

Reply:
Yes, the links would be very useful. Also add a relevant OpenMetrics link.